src/views/Viewer.vue

```diff
@@ -351,12 +351,14 @@ export default {
 		subscribe('files:sidebar:closed', this.handleAppSidebarClose)
 		window.addEventListener('keydown', this.keyboardDeleteFile)
 		window.addEventListener('keydown', this.keyboardDownloadFile)
+		window.addEventListener('keydown', this.keyboardEditFile)
 	},
 
 	beforeDestroy() {
 		window.removeEventListener('resize', this.onResize)
 		window.removeEventListener('keydown', this.keyboardDeleteFile)
 		window.removeEventListener('keydown', this.keyboardDownloadFile)
+		window.removeEventListener('keydown', this.keyboardEditFile)
```

Comment:
Wouldn't it be cleaner to have one method for the keyboardListener?
https://github.com/nextcloud/viewer/blob/05de35dd3ef2425960543c82310a5ee2a807fa1d/src/components/ImageEditor.vue#L161-L182

Reply:
probably yes? Shall I change this?

Reply:
I think both is fine...

Reply:
Ok, lgtm then :)